### PR TITLE
Women in tech

### PR DESCRIPTION
**Pull request by rudri182**
It is issue number #706. Please review it and suggest how to resolve conflicts

### CODE REVIEW

**Comment by Kundan28** [comment]
@rudri182 Please address the issue number in the comments as it's easy for the reviewers to review the PR.

**Comment by vsharma20v** [comment]
@rudri182  the markdown of the link is not correct it should be
- [Inspiring the next generation of female engineers | Debbie Sterling | TEDxPSU ](https://youtu.be/FEeTLopLkEo)

**Comment by rudri182** [comment]
Committed changes directly on Github. Are there any more changes to be done? How can I push these changes?

**Comment by Kundan28** [comment]
@rudri182 Resolve the conflicts.

**Comment by rudri182** [comment]
What are the other conflicts that are still remaining?

**Comment by rudri182** [comment]
Is this PR ready to merge? @Kundan28 If there are still any conflicts, I am not able to identify what are they. 
Please mention those and I will resolve it.

File: women-in-tech-resources/resources-list.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ Here are links to lessons that should be completed before this lesson.
 - [Networking](/networking/networking.md)
 - [Diversity, Inclusion, and Bias](/diversity-inclusion-bias/inclusion.md)
 
+### Motivation
+- (Inspiring the next generation of female engineers|Debbie Sterling|TEDxPSU)[https://youtu.be/FEeTLopLkEo]
```
[the remark this answers is not in the document]
Comment (alodahl) [reply women-in-tech-resources/resources-list.md]: Oh I see! Your brackets are wrong. Compare this to other links in the lesson.

**Comment by rudri182** [comment]
I have fixed the link in the motivation section. Thank you @alodahl 👍  for pointing out a very minor mistake. I was very confused that what are the conflicts that I am not able to figure out. Hope now conflicts are resolved.

**Comment by Kundan28** [comment]
@rudri182 
PR

**Comment by rudri182** [comment]
@Kundan28 is this now ready? Because when I am changing that file in [Resolve Conflicts](https://github.com/Techtonica/curriculum/pull/724/conflicts) it is saying that every conflict should be resolved. But I have made changes in the file and link is
https://github.com/Techtonica/curriculum/compare/403b8fdc48b7d05e4b74663226c1f452571929d0?expand=1.

**Comment by Kundan28** [comment]
It's showing 4 conflicts. Go through this below link.
https://github.com/Techtonica/curriculum/pull/724/conflicts

**Comment by rudri182** [comment]
> It's showing 4 conflicts. Go through this below link.
> https://github.com/Techtonica/curriculum/pull/724/conflicts

I made changes like
<<<<<<< women-in-tech
- [Inspiring the next generation of female engineers|Debbie Sterling|TEDxPSU](https://youtu.be/FEeTLopLkEo)
=======
- [Inspiring the next generation of female engineers|Debbie Sterling|TEDxPSU](https://youtu.be/FEeTLopLkEo)
>>>>>>> master
but I am not able to do 'mark as resolved'

**Comment by Kundan28** [comment]
Delete the conflict markers <<<<<<<, =======, >>>>>>> and make the changes you want in the final merge.

**Comment by rudri182** [comment]
image

@Kundan28

**Comment by Kundan28** [comment]
There are more conflicts below you are only solving the first one I think.

**Comment by rudri182** [comment]
> There are more conflicts below you are only solving the first one I think.

May I remove all <<<<,====,>>>> ? From below also ? Are they conflicting?

**Comment by Kundan28** [comment]
Yeah, they too are conflicting. And according to me yes you have to remove all <<<<,====,>>>>

**Comment by Kundan28** [comment]
Congratulations @rudri182 🔥

**Comment by rudri182** [comment]
All the conflicts resolved finally. 👍

**Comment by rudri182** [comment]
> I hope you understood how to resolve conflicts. 😃

Yes. Because it is the first time I am resolving so many conflicts

**Comment by Kundan28** [comment]
@alodahl Not this PR is ready to merge.

**Comment by Kundan28** [comment]
> Yes. Because it is the first time I am resolving so many conflicts

That's great.
Today you have learned something new.

**Comment by rudri182** [comment]
> > Yes. Because it is the first time I am resolving so many conflicts
> 
> That's great.
> Today you have learned something new.

Yes definitely.

**Comment by alodahl** [comment]
@rudri182 👏🏼👩‍🎓🏅

**Pull request merged**